src/ui/Table/Table.css

```diff
@@ -68,3 +68,18 @@
 .tableui [data-expanded='false'] {
   @apply hidden;
 }
+
+/* Scrollbar styling */
```

Comment:
nice! is this fixing the scrollbar issue @suejung-sentry was having?

Reply:
yes! it is fixed!